guides/common/modules/ref_managing-packages-on-the-base-operating-system.adoc

```diff
@@ -0,0 +1,55 @@
+[id="Managing_Packages_on_the_Base_Operating_System_{context}"]
+= Managing packages on the base operating system of {ProjectServer} or {SmartProxyServer}
+
+{Project} provides the `{foreman-maintain} packages` command to install and update packages on the base operating system of your {ProjectServer} or {SmartProxyServers}.
+
+[WARNING]
+====
+Installing and updating individual packages with `yum` is not supported.
+While {Project} provides the `{foreman-maintain} packages unlock` command to bypass `{foreman-maintain} packages` and enable package management with `yum`, this can lead to package inconsistencies and compromise the stability of the system.
+Always use `{foreman-maintain} packages` to manage packages.
+====
+
+After you install or update a package on the base operating system, running `{foreman-installer}` afterwards is required.
+When you manage packages with `{foreman-maintain} packages`, the command automatically runs `{foreman-installer}` as well.
+Note that this results in restarting some services on the base operating system.
```

Comment:
```suggestion
Note that this results in restarting any required services on the base operating system.
```

"some" feels very unspecific. Is "any required" any better? Or maybe "restarting affected services"?

Reply:
I think I like "affected services", thanks!